chingu-voyage5/Bears-Team-5
# Chore/#32 express server

closes #32 
closes #33 

## Changes
* Setup express server
* Setup middleware
  * Included morgan, helmet, cookie-parser, express-sessions, and cors
* Updated readme scripts

## Testing
* Update your `.env` file with the `PORT` and `SESSION_SECRET`
* `npm install` new dependencies
* Run server with `npm run server` or `npm run dev-server` (nodemon)
* In browser, go to `localhost:3001` to see testing message (to be deleted in the future)

## Notes
* Couldn't get a separate `package.json` setup properly in `server` directory
